src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => value.split(':').map((spaces) => spaces.trim()))
```

Comment:
The `.map()` call within the chain is used to trim spaces from the CSS property and value. This is good, but it is important to ensure that the split(':') call is handling cases where there might be multiple colons in the property values. While not required for this task, it would be a nice addition to handle such edge cases for a more robust function.